ansible_ai_connect/main/settings/base.py

```diff
@@ -582,3 +585,38 @@
 ANSIBLE_AI_ENABLE_ONE_CLICK_DEFAULT_MODEL_ID: str = (
     os.getenv("ANSIBLE_AI_ENABLE_ONE_CLICK_DEFAULT_MODEL_ID") or ""
 )
+
+t_one_click_reports_postman_type = Literal[
+    "none", "stdout", "slack-webhook", "slack-webapi", "google-drive"
+]
+ANSIBLE_AI_ONE_CLICK_REPORTS_POSTMAN: t_one_click_reports_postman_type = os.getenv(
+    "ANSIBLE_AI_ONE_CLICK_REPORTS_POSTMAN"
+) or cast(t_one_click_reports_postman_type, "none")
+ANSIBLE_AI_ONE_CLICK_REPORTS_SLACK_TOKEN: str = (
+    os.getenv("ANSIBLE_AI_ONE_CLICK_REPORTS_SLACK_TOKEN") or ""
+)
+ANSIBLE_AI_ONE_CLICK_REPORTS_SLACK_CHANNEL_ID: str = (
+    os.getenv("ANSIBLE_AI_ONE_CLICK_REPORTS_SLACK_CHANNEL_ID") or ""
+)
+ANSIBLE_AI_ONE_CLICK_REPORTS_SLACK_WEBHOOK_URL: str = (
+    os.getenv("ANSIBLE_AI_ONE_CLICK_REPORTS_SLACK_WEBHOOK_URL") or ""
+)
+ANSIBLE_AI_ONE_CLICK_REPORTS_GOOGLE_DRIVE_FOLDER_NAME: str = (
```

Comment:
can we have just one env var expecting a json string to include all these stuff?

Reply:
OK. Done.